src/main/resources/xsd/Questionnaire.xsd

```diff
@@ -658,6 +672,63 @@
     </xs:sequence>
   </xs:complexType>
 
+  <xs:complexType name="NomenclatureType">
+    <xs:sequence>
+      <xs:element name="name" type="xs:string"/>
+      <xs:element name="fields" type="SuggesterField" minOccurs="1" maxOccurs="unbounded"/>
+      <xs:element name="max" type="xs:integer" minOccurs="0"/>
+      <xs:element name="stopWords" type="xs:string" minOccurs="0" maxOccurs="unbounded"/>
+      <xs:element name="order" type="SuggesterOrder" minOccurs="0"/>
+      <xs:element name="queryParser" type="SuggesterQueryParser"/>
+      <xs:element name="url" type="xs:string" minOccurs="0"/>
+      <xs:element name="version" type="xs:string"/>
+    </xs:sequence>
+    <xs:attribute name="id" type="xs:ID" use="required"/>
+  </xs:complexType>
+
+  <xs:complexType name="SuggesterField">
+    <xs:sequence>
+      <xs:element name="name" type="xs:string"/>
+      <xs:choice>
```

Comment:
"rules" and "rulesA" shouldn't exist both as they have the same purpose.

Best way i think is to make "rules" an array of string and get rid of "rulesA". If i understand correctly of how you want to make it work: 
- if the attribute doesn't exist, apply a default value (array of strings with X values)
- if the attribute exists, the format should be an array of strings with X values